src/api.py

```diff
@@ -378,15 +338,29 @@ def _flagged(messages: List[Dict[str, str]]) -> bool:
         moderation = litellm.moderation(input=input_text)
         return moderation.results[0].flagged
 
+    @staticmethod
+    def _validate_options(options: Dict[str, Any]) -> dict:
+        # TODO PR: Block Replicate here?  If delegated user isn't steamship?  Because of known billing bug
```

Comment:
We don't currently bill correctly (or at least haven't verified newer versions of LiteLLM) for replicate.  We can't wholesale block it here and raise if we want to use it internally, but we could check against user, which we decide from the engine side via auth?

Reply:
Do we want to just make the plugin private for now?  This would eliminate the possibility that anyone else could use it, and then any billing issues would just be a problem for our own accounting.

Reply:
Yes -- the plugin should be private and only Steamship-accessible for now.

Reply:
It is private for now; this was a suggestion to be defensive so that we don't have to remember to fix this if we ever decide to go public with it.